patches/net/minecraft/world/level/block/AbstractCauldronBlock.java.patch

```diff
@@ -0,0 +1,24 @@
+--- a/net/minecraft/world/level/block/AbstractCauldronBlock.java
++++ b/net/minecraft/world/level/block/AbstractCauldronBlock.java
+@@ -102,4 +_,21 @@
+ 
+     protected void receiveStalactiteDrip(BlockState p_151975_, Level p_151976_, BlockPos p_151977_, Fluid p_151978_) {
+     }
++
++    @Override
++    public void onPlace(BlockState p_51978_, Level p_51979_, BlockPos p_51980_, BlockState p_51981_, boolean p_51982_) {
++        // Neo: Invalidate cauldron capabilities when a cauldron is added
```

Comment:
Similar to the onRemove call we may as well call super here so that in case there ever is default code added we don't have to worry about it when porting.